sys-kernel/coreos-modules/files/commonconfig-4.14

```diff
@@ -46,6 +46,7 @@ CONFIG_MODULE_UNLOAD=y
 CONFIG_MODULE_SIG=y
 CONFIG_MODULE_SIG_SHA256=y
 CONFIG_BLK_DEV_THROTTLING=y
+CONFIG_BLK_WBT=y
```

Comment:
Should we be enabling `BLK_WBT_SQ` as well?

Reply:
Well, I do not know. A few notes about BLK_WBT_SQ:
* It is supposed to enable writeback throttle by default on single-queue devices. However, it did not work for me. 
* Since writeback throttle can be enabled for any device (regardless of BLK_WBT_SQ) via a simple udev rule, instead of debugging why BLK_WBT_SQ did not work, I simply used a udev rule to enable writeback throttle, using ignition like this:

```
    - path: /etc/udev/rules.d/60-wbt-enable.rules
      filesystem: root
      mode: 0644
      contents:
        inline: |
          ACTION=="add|change", KERNEL=="sd[b-z]*", ATTR{queue/wbt_lat_usec}="75000"
```